### PR TITLE
chore: add files

### LINKED ISSUE #1: Move docs to this repo
Cleanup docs in: http://help-dot-itsl-pilot.appspot.com/api/organisations/index.html
Remove the JS stuff, keep it only to the general API description of the REST and websocket API.
Also update docs where needed (remove organisation from the URL)
You can also assume ownership of this commit: https://build.d-centralize.nl/gerrit/#/c/11087/4/docs/api/authentication.md

### PR DESCRIPTION
**Pull request by Lehren**
Add all files needed to render docs.

Resolve #1

### CODE REVIEW

File: app.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+service: help
```
Comment (kvdb) [inline app.yaml]: this file is no longer needed on github

File: index.html [hunk 2]
```diff
@@ -0,0 +1,346 @@
+<!DOCTYPE html>
```
Comment (kvdb) [inline index.html]: this is a generated file, please remove from repo

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+setup(
+    name='itslanguage-docs',
+    author='d-centralize',
+    maintainer='d-centralize',
+    version='0.0.0',
+    packages=find_packages(),
+    install_requires=[
+        'aafigure==0.5',
+        'mkdocs==0.14.0',
+        'webapp2==2.5.2'],
```
Comment (kvdb) [inline setup.py]: webapp2 is no longer needed

**Review by kvdb — CHANGES_REQUESTED** [review]

**Comment by Lehren** [comment]
Fixed

**Review by kvdb — APPROVED** [review]

**Pull request merged**